roles/picksome/files/picksome.php

```diff
@@ -0,0 +1,67 @@
+<?php
+# While this next line should be taken care of in extenions, and indeed
```

Comment:
I found this sentence somewhat hard to parse :-).

Reply:
I cleaned this up, and the following too.